pkg/features/supported.go

```diff
@@ -24,6 +25,7 @@ var (
 
 	// Dump/Restore
 	DumpMiddleware    = plugins.Feature[types.Middleware[types.Dump]]{Symbol: "DumpMiddleware", Description: "Dump middleware"}
+	DumpVMMiddleware  = plugins.Feature[types.Middleware[types.DumpVM]]{Symbol: "DumpVMMiddleware", Description: "Dump VM middleware"}
 	RestoreMiddleware = plugins.Feature[types.Middleware[types.Restore]]{Symbol: "RestoreMiddleware", Description: "Restore middleware"}
```

Comment:
Will have to add `DumpVMHandler` and same for restore as each VM plugin will implements its own.